libs/audit-service/src/main/java/uk/gov/di/ipv/core/library/auditing/extension/AuditExtensionsUserIdentity.java

```diff
@@ -17,7 +18,7 @@
 @NoArgsConstructor
 @AllArgsConstructor
 public class AuditExtensionsUserIdentity implements AuditExtensions {
-    @JsonProperty String levelOfConfidence;
+    @JsonProperty Vot levelOfConfidence;
```

Comment:
(I assume this 'just works' for serialization)

Reply:
Yep - I've checked the audit event logs and it's happy.